src/main/java/com/google/api/gax/grpc/ApiCallSettingsTyped.java

```diff
@@ -15,26 +25,29 @@
     return new DirectCallable<>(clientCallFactory);
   }
 
-  ApiCallSettingsTyped(Builder<RequestT, ResponseT> builder) {
-    super(builder);
-    this.methodDescriptor = builder.getDescriptor();
+  ApiCallSettingsTyped(Set<Status.Code> retryableCodes,
```

Comment:
Add a note about why this is package-private. (package-private should always be documented, because it's not an explicit access modifier and thus easy to miss)